apps/condo/domains/property/components/panels/Builder/BuildingPanelCommon.tsx

```diff
@@ -109,3 +208,57 @@ export function useHorizontalScroll (): React.RefObject<HTMLElement>{
     }, [])
     return elementRef
 }
+
+
+const getBrowserFullscreen = (): string | null => {
+    let result
+
+    if (typeof document.fullscreenElement !== 'undefined') {
+        result = 'fullscreenElement'
+    } else if (typeof document['mozFullScreenElement'] !== 'undefined') {
+        result = 'mozFullScreenElement'
+    } else if (typeof document['webkitFullscreenElement'] !== 'undefined') {
+        result = 'webkitFullscreenElement'
+    } else {
+        throw new Error('This browser not supports fullscreenElement')
+    }
+    return result
+}
+
+const requestFullscreen = async (elem): Promise<void> => {
+    if (elem.requestFullscreen) {
+        await elem.requestFullscreen()
```

Comment:
Are you sure that we need browser level full screen ? As for me we just need to change display view